AugmentedNet/annotation_parser.py

```diff
@@ -131,10 +131,12 @@ def _extractRomanNumeralInformation(rn):
         and "Fr" not in rn.figure
         and "It" not in rn.figure
     ):
-        if rn.figure in ["I4", "iv4", "v4", "V4"]:
-            rn.figure = rn.figure.replace("4", "")
+        if rn.figure in ["I4", "iv4", "v4", "V4", "I54", "i54"]:
+            rn.figure = rn.figure.replace("5", "").replace("4", "")
         if rn.figure in ["V9", "V7M9"]:
             rn.figure = "V7"
+        if rn.figure == "iio7":
+            rn.figure = "iiø7"
```

Comment:
This is only appropriate in major, and when there's no mixture. I may be missing something but the surrounding code doesn't seem to make that distinction.

Reply:
Thank you for bringing this up. I need to think a bit more about this one, esp. with examples. The way I understand this, `iio7` only makes sense if you are saying something like `viio7/ii` but somehow deciding to indicate it as `iio7` instead of `viio7/ii`. In any other instance, `iiø7` is the diatonic construction of the seventh of the second degree of a minor key, which is often a subdominant going to `V` `i`. For example: `c: i iiø7 V7 i`. Is my assumption wrong?

Reply:
The first instance I am referring to (a correct `iio7` annotation that I am effectively contaminating with my code) would be something like `C: i bVII iio7 III`, which I expect would be the minority of cases.

Reply:
Oh right, sorry yes I get it now. My mistake. I'd be interested to see the examples.

Reply:
For what is worth, I verified the objective metrics after doing this change, about 10 pieces changed and only in one it seemed to change for worse. Maybe I need to look more into the specifics of these pieces.